Makefile

```diff
@@ -38,7 +41,7 @@ test_cov:
 compile_env:
 	poetry lock --no-update
 
-build: isort black pre_commit flake8 mypy test
+build: isort black pre_commit flake8 mypy vulture test
```

Comment:
Github's build uses the pre-commit for convenience reasons. Once we remove _vulture_ from pre-commit, we also remove it from the build pipeline. This creates a discrepancy as CI/CD build will differ from `make build`. It's the same case as with _mypy_. We should add _vulture_ separately to the build workflow.